### PR TITLE
Dropdown :droplet: 

### LINKED ISSUE #24: Create dropdown

### PR DESCRIPTION
**Pull request by alexandrelam**
dropdown

### CODE REVIEW

File: components/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+import { ICheck, IChevronBottom } from './icons';
+import useOnClickOutside from '../hooks/useOnClickOutside';
+
+export type Options = {
+  label: string;
+  value: string;
```
Comment (Meltical) [inline components/Dropdown.tsx:7]: Maybe use generics for value ?

File: pages/groups.tsx [hunk 2]
```diff
@@ -2,8 +2,55 @@ import Header from '../components/Header';
 import { IGroup } from '../components/icons';
 import NewButton from '../components/NewButton';
 import Banner from '../components/Banner';
+import Dropdown, { Options } from '../components/Dropdown';
+import { useState } from 'react';
 
 export default function Groups(): JSX.Element {
+  const optionsGenre = [
+    {
+      label: 'Rock',
```
Comment (RomainGuarinoni) [inline pages/groups.tsx:11]: In the DB, it''s name and not label

**Review by RomainGuarinoni — APPROVED** [review]
LGTM !!

File: pages/groups.tsx (hunk 2 above)
Comment (alexandrelam) [reply pages/groups.tsx:11]: same shit, default select have `label` as a naming convention

**Pull request merged**